woocommerce/woocommerce-android
# Update order analytics for metadata

Summary
==========
Fix issue #6939 by adjusting the `ORDER_OPEN` event in the Order list view to set additional info to the track event:

- custom_fields_count: describing the amount of the metadata available in that order
- custom_fields_size: describing the total size in bytes of all metadata value fields combined

How to Test
==========
1. Simply select an Order in the Order List view that contains metadata on it, and verify in the logs that the `ORDER_OPEN` track event includes the expected new fields 

```id I/WooCommerce-UTILS: 🔵 Tracked: order_open, Properties: {"id":118,"status":"pending","custom_fields_count":2937,"custom_fields_size":19,"blog_id":204071362,"is_wpcom_store":true,"is_debug":true}```

Update release notes:
